### PR TITLE
Add Hotword docs for sherpa-onnx

### PR DESCRIPTION
**Pull request by pkufool**

### CODE REVIEW

File: docs/source/onnx/hotwords/index.rst [hunk 1]
```diff
@@ -0,0 +1,919 @@
...
+   git lfs pull --include "*.onnx"
+   git lfs pull --include "bpe.model"
+   cd ..
+   ln -s sherpa-onnx-zipformer-en-2023-04-01 exp
+
+Convert the hotwords into tokens
+
+.. code-block::
+
+   sherpa-onnx text2token \
```
Comment (csukuangfj) [inline docs/source/onnx/hotwords/index.rst:320]: `sherpa-onnx` is also a binary.
There would be a conflict for such a design.

File: docs/source/onnx/hotwords/index.rst [hunk 2]
```diff
@@ -0,0 +1,919 @@
...
+**Download the model**
+
+.. code-block:: bash
+
+   cd /path/to/sherpa-onnx
+
+   GIT_LFS_SKIP_SMUDGE=1 git clone https://huggingface.co/csukuangfj/sherpa-onnx-zipformer-en-2023-04-01
+   cd sherpa-onnx-zipformer-en-2023-04-01
+   git lfs pull --include "*.onnx"
+   git lfs pull --include "bpe.model"
```
Comment (csukuangfj) [inline docs/source/onnx/hotwords/index.rst:312]: could you upload `bpe.model` to this repo?

File: docs/source/onnx/hotwords/index.rst [hunk 3]
```diff
@@ -0,0 +1,919 @@
...
+.. code-block::
+
+    python python-api-examples/offline-decode-files.py \
+        --encoder=exp/encoder-epoch-99-avg-1.onnx \
+        --decoder=exp/decoder-epoch-99-avg-1.onnx \
+        --joiner=exp/joiner-epoch-99-avg-1.onnx \
+        --decoding=modified_beam_search \
+        --tokens=exp/tokens.txt \
+        --hotwords-file hotwords_en.txt \
+        --hotwords-score 2.0 \
```
Comment (csukuangfj) [inline docs/source/onnx/hotwords/index.rst:486]:
```suggestion
        --hotwords-file=hotwords_en.txt \
        --hotwords-score=2.0 \
```
Please use a consistent style.

**Review by csukuangfj — CHANGES_REQUESTED** [review]

File: docs/source/onnx/hotwords/index.rst (hunk 1 above)
Comment (pkufool) [reply docs/source/onnx/hotwords/index.rst:320]: can we change to `sherpaonnx`? or `sherpa-onnx-cli` ?

File: docs/source/onnx/hotwords/index.rst (hunk 2 above)
Comment (pkufool) [reply docs/source/onnx/hotwords/index.rst:312]: I think I already made a pull request

File: docs/source/onnx/hotwords/index.rst (hunk 1 above)
Comment (pkufool) [reply docs/source/onnx/hotwords/index.rst:320]: In that case, I will make it a command line tool and script in the repo, I think I once asked you whether people is able to use sherpa-onnx without cloneing our repo, you said yes. So I think we should make it a command tool.

File: docs/source/onnx/hotwords/index.rst (hunk 2 above)
Comment (csukuangfj) [reply docs/source/onnx/hotwords/index.rst:312]: Screenshot 2023-09-13 at 10 22 51

There are no pull-requests at all.

Comment (pkufool) [reply docs/source/onnx/hotwords/index.rst:312]: Ah, I think I made one here (https://huggingface.co/csukuangfj/sherpa-onnx-streaming-zipformer-bilingual-zh-en-2023-02-20) but not https://huggingface.co/csukuangfj/sherpa-onnx-zipformer-en-2023-04-01, will do it when huggingface is available.

**Pull request merged**